Documentation/release-notes/cpp-aab.md

```diff
@@ -0,0 +1,5 @@
+#### C++ projects
+
+* [GitHub Issue 4998](https://github.com/xamarin/xamarin-android/issues/4998):
+  *Files under lib/ must have .so extension, found 'lib/x86/gdbserver'.* build error
+  could prevent C++ projects from building app bundles.
```

Comment:
Looks great on formatting. For wording, if I'm skimming the issue correctly, I think the error happens when attempting to build an app bundle for a Xamarin.Android app project that references a C++ Android library project, so maybe something like:

```
* [GitHub 4998](https://github.com/xamarin/xamarin-android/issues/4998):
  *Files under lib/ must have .so extension, found 'lib/x86/gdbserver'.* build error
  prevented building app bundles for projects with references to Android C++ library
  projects.
```